microsoft/testsuites/lis/lissuite.py

```diff
@@ -242,6 +233,19 @@ def _check_lis_version_hex(
             f" LIS version: {source_version_hex}, Actual LIS version: {version_hex}"
         )
 
+    def _install_lis(self, node: Node) -> LisDriver:
+        try:
+            lisdriver = node.tools[LisDriver]
+            return lisdriver
+        except Exception as identifier:
+            if "cannot find [lisdriver]" in str(identifier):
```

Comment:
It's better to wrap the error in the LisDriver, and then raise an unsupported distro exception.

Reply:
updated